.eslintrc

```diff
@@ -0,0 +1,19 @@
+{
+  "extends": "airbnb",
```

Comment:
not sure if this has leaked through from this preset but I'm not a fan of requiring a dangling comma.